backend/api/migrations/0008_conversation_message.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.1.2 on 2018-12-04 20:59
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0007_auto_20181126_1400'),
```

Comment:
26 November 2018, at 14:00.

Voyager-1 is far more than 21.681.360.000 kilometers at that time. It's been more than 41 years since it's launch.

More: https://voyager.jpl.nasa.gov/mission/status/

Reply:
😮